### PR TITLE
App with extensions point: Move to Playwright and plugin-e2e

### PR DESCRIPTION
**Pull request by sunker**
This pr does the following:
* Removes cypress tests
* Removes grafana/e2e deps
* Adds plugin-e2e and playwright deps and config 
* Migrates existing Cypress tests to use Playwright and plugi-e2e

### CODE REVIEW

File: examples/app-with-extension-point/package.json [hunk 1]
```diff
@@ -5,8 +5,7 @@
   "scripts": {
     "build": "webpack -c ./.config/webpack/webpack.config.ts --env production",
     "dev": "webpack -w -c ./.config/webpack/webpack.config.ts --env development",
-    "e2e:update": "npm exec cypress install && npm exec grafana-e2e run --update-screenshots",
-    "e2e": "npm exec cypress install && npm exec grafana-e2e run",
+    "e2e": "npx playwright test",
```
Comment (mckn) [inline examples/app-with-extension-point/package.json:8]:
```suggestion
    "e2e": "playwright test",
```

I don't think we need to run it via npx

File: examples/app-with-extension-point/tests/extendActions.spec.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { test, expect } from '@grafana/plugin-e2e';
```
Comment (mckn) [inline examples/app-with-extension-point/tests/extendActions.spec.ts:1]: Was thinking about wether or not to keep the testIds. What do you think about replacing them with other selectors such as getByRole or getByLabel?

**Review by mckn — COMMENTED** [review]
LGTM! Added a couple of questions.

File: examples/app-with-extension-point/tests/extendActions.spec.ts (hunk 2 above)
Comment (sunker) [reply examples/app-with-extension-point/tests/extendActions.spec.ts:1]: There are no form elements where labels are associated with inputs. Could potentially use getByRole in some places, but I don't see any harm in using test ids here.

**Review by leventebalogh — APPROVED** [review]
LGTM 👍

**Pull request merged**